src/domain/queue/services/class.queue.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace leantime\domain\services {
+
+    use leantime\core;
+    use leantime\domain\repositories;
+
+    class queue
+    {
+
+        private $queue;
+        private $userRepo;
+        private $settingsRepo;
+        private $mailer;
+
+        public function __construct()
+        {
+            // NEW Queuing messaging system
+            $this->queue = new repositories\queue();
+
+            // We need users and settings and a mailer
+            $this->userRepo = new repositories\users();
+            $this->settingsRepo = new \leantime\domain\repositories\setting();
+            $this->mailer = new \leantime\core\mailer();
+        }
+
+        // Fake template to be replaced by something better
+        // TODO : Rework email templating system
+        private function doFormatMail ($messageToSendToUser)
+        {
+            $outputHTML="Leantime has news for you...<br/>\n";
```

Comment:
Instead of Leantime, should we use the `companysettings.sitename` setting, since we allow them to change it?

Reply:
What do you decide @marcelfolaron  ?